metrics/disk/cassandra_kubernetes/cassandra.sh

```diff
@@ -125,21 +125,51 @@ function cassandra_start() {
 	wait_time=20
  	sleep_time=2
 
+	vol_capacity="500M"
```

Comment:
Is this capacity enough for the cassandra workload? If I recall the stateful set was using at least 1G with three replicas. 
I would bump this to 3G atleast,

Reply:
@amshinde changes applied